apps/rod/sections/summary-data-sections.js

```diff
@@ -20,46 +20,59 @@ module.exports = {
       step: '/sponsor-type',
       field: 'sponsor-type'
     },
+    {
+      step: '/application',
+      field: 'application-type'
+    },
+    {
+      step: '/visa-type',
+      field: 'visa-type'
+    },
     {
       step: '/about-application',
       field: 'date-of-application',
       parse: d => d && dateFormater.format(new Date(d))
     },
+
     {
       step: '/about-application',
       field: 'cancel-application'
     },
     {
-      step: '/dependant-or-guardian',
-      field: 'dependant-or-guardian'
-    },
-    {
-      step: '/legal-representation',
-      field: 'legal-rep-name'
+      step: '/main-applicant-passport',
+      field: 'is-requesting-passport-to-travel'
     },
     {
-      step: '/application',
-      field: 'application-type'
+      step: '/further-leave',
+      field: 'further-leave-to-remain'
     },
     {
-      step: '/visa-type',
-      field: 'visa-type'
+      step: '/reference-number',
+      field: 'rod-reference-number',
+      parse: value => {
+        return Array.isArray(value) ? value.map(option => option).join('\n') : value;
```

Comment:
We need to put this information in the session. Please refer below PR/Change for the reference. 

https://github.com/UKHomeOffice/return-of-documents/pull/55/files#diff-ca36b2dc97bc512239e551233d9bb9d154d8ba1abe805116edd4d8b7d9355670